tests/rptest/tests/librdkafka_test.py

```diff
@@ -13,90 +13,64 @@
 from rptest.services.cluster import cluster
 from ducktape.mark import matrix, ignore
 
-from ducktape.tests.test import Test
-from rptest.services.redpanda import RedpandaService
+from rptest.services.librdkafka_test_case import LibrdkafkaTestcase
+from rptest.tests.redpanda_test import RedpandaTest
 
+from rptest.utils.mode_checks import skip_debug_mode
 
-class LibrdkafkaTest(Test):
+
+class LibrdkafkaTest(RedpandaTest):
     """
     Execute the librdkafka test suite against redpanda.
     """
     TESTS_DIR = "/opt/librdkafka/tests"
     CONF_FILE = os.path.join(TESTS_DIR, "test.conf")
 
     def __init__(self, context):
-        super(LibrdkafkaTest, self).__init__(context)
-        self._context = context
-        self._extra_rp_conf = dict(
-            auto_create_topics_enabled=True,
-            default_topic_partitions=4,
-        )
-        self._redpanda = None
-
-    def _start_redpanda(self, num_brokers):
-        self._redpanda = RedpandaService(self._context,
-                                         num_brokers,
-                                         extra_rp_conf=self._extra_rp_conf)
-        self._redpanda.start()
-
-        with open(LibrdkafkaTest.CONF_FILE, "w") as f:
-            brokers = self._redpanda.brokers()
-            f.write("metadata.broker.list={}".format(brokers))
-
-    def teardown(self):
-        self._redpanda.stop()
-        os.remove(LibrdkafkaTest.CONF_FILE)
+        super(LibrdkafkaTest,
+              self).__init__(context,
+                             num_brokers=3,
+                             extra_rp_conf={
+                                 "enable_leader_balancer": False,
```

Comment:
Any specific reason for `enable_leader_balancer` to be False? ERROR logs?

Since this is not default, we are not testing the default RP configuration that external users may use to test with librdkafka. Consider removing it if possible.

Reply:
Actually this flag can be deleted.